rcsb/db/cli/RepoLoadExec.py

```diff
@@ -95,6 +96,8 @@ def main():
     parser.add_argument("--num_sublists", default=None, help="Number of sublists to create/load for the associated database")
     parser.add_argument("--force_reload", default=False, action="store_true", help="Force re-load of provided ID list (i.e., don't just load delta; useful for manual/test runs).")
     parser.add_argument("--provider_types_exclude", default=None, help="Resource provider types to exclude")
+    parser.add_argument("--outputContentType", action="store_true", default=False, help="Whether output path in downstream application has prepended content type (pdb, csm) before file name")
+    parser.add_argument("--outputHash", action="store_true", default=False, help="Whether output path in downstream application has prepended hash before file name")
```

Comment:
Can you change the argument to be snakecase instead of camel, just to be consistent with the other CLI argument names? (I know, it's annoying that these CLI ones are snake while everything else is camel, but it's too late to change everything now...)

Reply:
Also, since these are flags, I might suggest prepending both arguments/variables with `prepend`, to make it more clear that they are flags and not input parameters, e.g.:

```
--prepend_output_content_type (corresponding var: prependOutputContentType)
--prepend_output_hash         (corresponding var: prependOutputHash)
```